packages/front-end/components/Auth/CreateOrJoinOrganization.tsx

```diff
@@ -15,9 +16,13 @@ import Field from "@/components/Forms/Field";
 import LoadingOverlay from "@/components/LoadingOverlay";
 import WelcomeFrame from "./WelcomeFrame";
 
-import style from "./CreateOrganization.module.scss";
+import style from "./CreateOrJoinOrganization.module.scss";
 
-export default function CreateOrganization(): ReactElement {
+const CreateOrJoinOrganization: FC<{
+  showFrame?: boolean;
+  title?: string;
+  subtitle?: string;
+}> = ({ showFrame = true, title = "", subtitle = "" }) => {
```

Comment:
its safer to leave these as undefined here for the title and subtitle for your check later on to see if it is false.